src/everest/jobs/io/__init__.py

```diff
@@ -1,22 +1,22 @@
 import json
 import os
+from typing import Any
 
 from ruamel.yaml import YAML
 
 
-def load_data(filename):
+def load_data(filename: str) -> Any:
```

Comment:
I might be wrong here but I would expect both `YAML(typ="safe", pure=True).load(fin)` and `json.load(fin)` will return `dict[str, Any]`, you could try using `dict[str, Any]` as a return type I know it is not much better than `Any` but it is something.

Reply:
Both YAML.load() and json.load() has Any return type

Reply:
You might be right, I just assumed that the return type would be `dict[str, Any]` after trying this:

```
d = {"a": 1}
import json
with open("test", "w", encoding="utf-8") as f:
    json.dump(d, f)

Path("test").read_text(encoding="utf-8")
Out[22]: '{"a": 1}'
with open("test", mode="r", encoding="utf-8") as f:
    x = json.load(f)
type(x)
Out[24]: dict
```

```
import yaml
with open("test_yml", mode= "w", encoding="utf-8") as f:
    yaml.dump(d, f)
Path("test_yml").read_text()
Out[27]: 'a: 1\n'
with open("test_yml", mode="r", encoding="utf-8") as f:
    x_yml = yaml.safe_load(f)
type(x_yml)
Out[29]: dict
```